docs/core/tools/global-json.md

```diff
@@ -1,32 +1,96 @@
 ---
-title: global.json reference
-description: See the schema for the global.json file, which permits setting the .NET Core tools version.
-author: blackdwarf
+title: global.json overview
+description: Learn how to use the global.json file to set the .NET Core SDK version when running .NET Core CLI commands.
+author: mairaw
 ms.author: mairaw
-ms.date: 04/05/2017
+ms.date: 06/28/2018
+ms.custom: "updateeachrelease"
 ---
-# global.json reference
+# global.json overview
 
-The *global.json* file allows selection of the .NET Core tools version being used through the `sdk` property.
+[!INCLUDE [topic-appliesto-net-core-all](../../../includes/topic-appliesto-net-core-all.md)]
 
-.NET Core CLI tools look for this file in the current working directory (which isn't necessarily the same as the project directory) or one of its parent directories.
+The *global.json* file allows you to define which .NET Core SDK version is used when you run .NET Core CLI commands.
+
+.NET Core SDK looks for a *global.json* file in the current working directory (which isn't necessarily the same as the project directory) or one of its parent directories.
+
+## global.json schema
+
+### sdk
 
-## sdk
 Type: Object
 
-Specifies information about the SDK.
+Specifies information about the .NET Core SDK to select.
+
+#### version
 
-### version
 Type: String
 
-The version of the SDK to use.
+The version of the .NET Core SDK to use.
 
-For example:
+Note that this field:
+
+- Doesn't have globbing support, that is, the full version number has to be specified.
+- Doesn't support version ranges.
+
+The following example shows the contents of a *global.json* file:
 
 ```json
 {
   "sdk": {
-    "version": "1.0.0-preview2-003121"
+    "version": "2.1.300"
   }
 }
 ```
+
+## global.json and the .NET Core CLI
+
+It's helpful to know which versions are available in order to set one in the *global.json* file. You can find the full list of supported available SDKs at the [.NET Downloads](https://www.microsoft.com/net/download/all) site. Starting with .NET Core SDK 2.1, you can run the following command to verify which SDK versions are already installed on your machine:
+
+```console
+dotnet --list-sdks
+```
+
+To install additional .NET Core SDK versions on your machine, visit the [.NET Downloads](https://www.microsoft.com/net/download/all) site.
+
+You can create a new the *global.json* file in the current directory by executing the [dotnet new](dotnet-new.md) command, similar to the following example:
+
+```console
+dotnet new globaljson --sdk-version 2.1.300
+```
+
+## Matching rules
+
+Starting with .NET Core SDK 2.0, the following rules apply when determining which version of the SDK to use:
```

Comment:
Sure I can. So me let me confirm. If I have 1.1 SDK and then install 2.1 runtime only on my machine, when trying to resolve the SDK specified on global.json, it will use this logic and not the 1.1 logic. I might have to add a note about this then because it’s counterintuitive.

Reply:
Rats. You brought up 1.1 SDK, which I am woefully unfamiliar with 

@dsplaisted can you clarify this?

Reply:
The version selection logic is part of the "Host" which is part of the .NET Core Runtime, but the latest version of the host is used when you have multiple Runtimes installed side-by-side.

So I think this can be worded as:

> Starting with .NET Core 2.0, the following rules apply...